vip_cram.nf

```diff
@@ -53,7 +53,7 @@ workflow cram {
     // merge outputs of snv, str and sv workflows
     Channel.empty().mix(ch_cram_snv, ch_cram_str, ch_cram_sv)
       | map { meta, vcf -> [groupKey(meta, nrActivateVariantCallerTypes), vcf] }
-      | groupTuple(remainder: true, sort: { left, right -> left.index <=> right.index })
+      | groupTuple(remainder: true)
```

Comment:
This breaks if the vcf is null, probably also a bug in production of one of the callers yields no results.
Why did we sort here? could not find the reason for this.

Reply:
> Why did we sort here?

Might have been a copy-paste error when applying `validateGroup` everywhere, see https://github.com/molgenis/vip/commit/c18c10873f9e43b6bf456e1e09281fc3e4fc282b#diff-6fd0867902770cb440cfbd68f89c59b518024cc178808ecb27a002b62b24c111R50